### DISCUSSION TITLE
trying to use world.shiftOrigin to scroll a scene

### DISCUSSION THREAD

**Discussion by bobcgausa** [shakiba/planck.js#193] [Q&A]
never mind, the testbed just uses translate
so I change my library to scroll the view the same way

**Comment by shakiba** [comment]
If you can share an example we may be able to help.

**Comment by bobcgausa** [comment]
Mr ShakibaThank you so much for replying.Never mind.I just defined a world view box and an origin and use graphics translate.It has secondary advantage in that I destroy objects that go out of bounds.Bob CookPs I will soon have a web site to share for p5js and Planck examples with my libraryOn May 10, 2021 9:18 PM, Ali Shakiba ***@***.***> wrote:
If you can share an example we may be able to help.
